### ISSUE TITLE
non-model species

### ISSUE THREAD

**Issue by andreaschavez** [nservant/HiC-Pro#575]
Hi:
I am studying a non-model diploid mammal species with a large genome size (~6GB). I have 30X Hifi data, as well as Hi-C data from Dovetail Genomics using their Chicago libraries. I have made an assembly in HiFiASM with the HiFi data. I would like to use EndHiC to scaffold my assembly with Hi-C data. EndHiC recommends using Hi-C Pro to generate input files. Hi-C Pro wants a table file of chromosome sizes. I don't have chromosome size information for my species. Do you know if I can create a table of scaffold lengths from my HiFiasm assembly? Or do these methods only really work on species with already known genome information? Thanks in advance. 
best,
Andreas

**Comment by SwiftSeal** [comment]
Did you find a solution to this? I'm struggling to understand the documentation on this.

Thanks!

**Comment by andreaschavez** [comment]
Hi SwiftSeal: I'm having similar struggles and haven't proceeded any further. I'm still interested in using the program if it seems feasible.

**Comment by SwiftSeal** [comment]
Yeah the config confused me a lot! In the end I just opted for yahs and manual correction with juicertools - had the issue of telomere fusions with yahs but not too tricky to correct manually.
